.github/workflows/run-k8s-tests-on-ppc64le.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
       DOCKER_REPO: ${{ inputs.repo }}
       DOCKER_TAG: ${{ inputs.tag }}
       PR_NUMBER: ${{ inputs.pr-number }}
+      GOPATH: ${{ github.workspace }}
```

Comment:
out of curiosity, why?
do we build something as part of this workflow?

Reply:
@fidencio, check this out - https://github.com/kata-containers/kata-containers/blob/main/tests/integration/kubernetes/filter_k8s_test.sh#L25-L31

This is mainly used on the non x86 arches to build/use `yq` which is used to filter out certain test cases.